docs/stories/14 - Views from Deck - Part 1.adoc

```diff
@@ -0,0 +1,60 @@
+:doctype: book
+:icons: font
+:page-background-image: image:background_neutral.jpg[fit=fill,pdfwidth=100%]
+
+= Views from Deck - Part 1
+Chris Garvey & Alex Jacobs
+
+_I turned to see a shore unlike anything I had ever known, and a child who beckoned me to come see this new land which opened before me._
+
+*From the journal of Hantei Botan, daughter of Emperor Hantei XXXVIII*
```

Comment:
We could format the whole section from the journal as quote, as follows:


```
[quote, From the journal of Hantei Botan, daughter of Emperor Hantei XXXVIII]
____
text
text
text
____
```

Reply:
If we do that, it looks like we lose the center alignment on the haiku... If we're good with that I can make that change.